### PR TITLE
move update settings item to trailing action

### PR DESCRIPTION
**Pull request by mateosalta**
- remove bottom anchored list item
- remove old divider
- place "Update Settings" as a trailing action in the header

### CODE REVIEW

**Review by dobey — APPROVED** [review]
Looks OK to me! Thanks.

**Comment by UniversalSuperBox** [comment]
For QA: Has "Update Settings" moved from the bottom of "System Settings -> Updates" to a cog icon in its header? If so, this is working.

File: plugins/system-update/PageComponent.qml [hunk 1]
```diff
@@ -366,16 +377,6 @@ ItemPage {
...
-        ListItem.ThinDivider {}
-
-        ListItem.SingleValue {
-            objectName: "configuration"
-            text: i18n.tr("Update settings")
-            progression: true
-            onClicked: pageStack.addPageToNextColumn(
-                root, Qt.resolvedUrl("UpdateSettings.qml"))
-        }
     }
```
Comment (mateosalta) [inline plugins/system-update/PageComponent.qml:380]: empty column left behind

**Review by mateosalta — COMMENTED** [review]
Clean up empty column, and anchor. https://github.com/mateosalta/system-settings/blob/d6e1bec457e48385a04d9fbe971f6676f3147fd3/plugins/system-update/PageComponent.qml#L96-L101

File: plugins/system-update/PageComponent.qml (hunk 1 above)
Comment (mateosalta) [reply plugins/system-update/PageComponent.qml:380]: https://github.com/mateosalta/system-settings/commit/6c5fd7f7190428e109764d01769f42da89709696

**Comment by cibersheep** [comment]
Awesome
tested on E5

**Comment by dadandalive** [comment]
OK for Nexus 5, RC 2021-W10

**Comment by ghost** [comment]
Tested on BQ Aquaris E4.5 and RC 2021-W10:
"Update settings" is available on the top under the cog icon. As expected.

**Pull request merged**